cefi/handler/capitalcom.py

```diff
@@ -61,6 +61,8 @@ def __init__(
                     pas=self.password,
                     api_key=self.api_key,
                 )
+
+            logger.debug("Client: {}", self.client)
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Potential sensitive information logging.

Logging the entire client object might inadvertently expose sensitive information. Consider logging only non-sensitive, identifying details.